ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -44,7 +44,7 @@ credentialsMode: Manual
 platform:
   azure:
     baseDomainResourceGroupName: openshiftInstallerRG
-    region: ${REGION}
+    region: ${AZURESTACK_REGION}
```

Comment:
As ENV `LEASED_RESOURCE` is defined as region on azure StackCloud, is it possible to set original variable `REGION` to `LEASED_RESOURCE`instead of introducing new ENV here?